docs/api/system-preferences.md

```diff
@@ -106,6 +106,15 @@ This is necessary for events such as `NSUserDefaultsDidChangeNotification`.
 
 Same as `unsubscribeNotification`, but removes the subscriber from `NSNotificationCenter`.
 
+### `systemPreferences.registerDefaults(defaults)` _macOS_
+
+* `defaults` Object - a dictionary of (`key: value`) user defaults
+  * `key` String - must be of type `string`
+  * `value` Object - object of type `string`, `boolean`, `integer`, `float`, `double`,
```

Comment:
1. Is it an `Object` or any of the following types?
2. There is no such types in JS as `integer`, `float`, `double`, `url`, or `dictionary` =/

Maybe it would be easier to just use a link to the `.getUserDefault` method like it's done in the `.setUserDefault()` description?

Reply:
If it can truly be any of those types it should be documented like so

```markdown
* `value` (String | Boolean | Number) - Description thing
```

Should check the typescript output to ensure it was picked up 👍